spec/spec_helper.rb

```diff
@@ -111,8 +111,15 @@
 
   config.include FactoryBot::Syntax::Methods
 
-  config.before { RequestHelper.stub_http_requests }
-  config.after { RequestHelper.real_http_requests }
+  config.before do
+    RequestHelper.stub_http_requests
+    Rails.application.load_seed
```

Comment:
this seems very expensive to run seeds before every test? shouldn't this happen once before the suite globally outside the transaction?